manifests/ncsa.pp

```diff
@@ -4,26 +4,21 @@
 #   enable service
 # @param service_ensure
 #   ensure serice
-# @param varnishncsa_daemon_opts
+# @param daemon_opts
 #   Options handed to varnishncsa
 class varnish::ncsa (
   Boolean $enable = true,
   Stdlib::Ensure::Service $service_ensure = 'running',
-  Optional[String] $varnishncsa_daemon_opts = undef,
+  String $daemon_opts = '-a -w /var/log/varnish/varnishncsa.log -D -P /run/varnishncsa/varnishncsa.pid',
```

Comment:
do we have to rename the parameter? That would be a breaking change.

Reply:
we don't have to.. but this class was not working since varnish 6, so quite a long time. So I thought cleanup would be ok